source/testnet/guides/add-baker-desktop.rst

```diff
@@ -0,0 +1,154 @@
+
+Export a transaction proposal
+=============================
+If more than one signature is needed to sign off on the baker account proposal, you must share a file of the type JSON with the co-signers. In the **Signatures** pane, you can see how many signatures are required before you can submit the transaction to the blockchain.
+
+#. In the Desktop Wallet, select **Export transaction proposal**.
+
+#. Navigate to the location on your computer where you want to save the file. In **Save as type** make sure that **All Files** is selected. Give the file a name and the extension .json, and then click **Save**.
+
+#. Send a copy of the file through a secure channel to the co-signers that must sign the transaction.
+
+Receive signatures from co-signers
+==================================
+
+When the co-signers have signed the transaction, they return the signed transaction proposal to you, and you have to import the files into the Desktop Wallet before you can submit the transaction to the chain.
+
+#. If you left the page with the account transaction, go to **Multi-signature Transactions**, and then select **Your proposed transactions**. If you're still on the same page, go to step 3.
+
+#. Select the transaction that you want to submit to the chain. You can see an overview of the transaction details and an overview of the signatures. You can also see that the status of the transaction is **Unsubmitted**, and you can see the identicon, and the transaction hash.
+
+#. Select **Browse to file** and then navigate to the location on your computer where you saved the signed transaction files. Select the relevant files, and then select **OK**. The files are uploaded to the Desktop Wallet and added to the list of signatures. Alternatively, you can drag and drop the signature files from their location on the computer and on to the Desktop Wallet.
+
+.. _submit-transaction:
+
+Submit the transaction to the blockchain
+========================================
+When you have received and added all the required signatures, you can submit the transaction to the blockchain.
+
+#. Review the transaction details carefully to ensure that all information is correct.
+
+#. Select **I understand this is the final submission, and that it cannot be reverted.**
+
+   -  If you don't want to submit the transaction to the chain, you can select **Close proposal**. The proposal is no longer active; however it is still visible in the list of proposals.
+
+#. Select **Submit transaction to chain.** The transaction is submitted to the chain and finalized on the Ledger.
+
+#. Select **Finish** to leave the page.
+
+.. Warning::
+    Transactions on the blockchain are permanent. That is, they are irreversible and can't be deleted them. Therefore, carefully review that you have selected the right account to add as baker, and that you have entered the correct amount to stake.
```

Comment:
"That is, they are irreversible and can't be deleted them."  I think "them" is maybe a typo?